### ISSUE TITLE
Keyboard focus is overlapping with content of the link

### ISSUE THREAD

**Issue by lovakumar** [OfficeDev/office-ui-fabric-react#4945]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: 
OS: Windows 10 RS3
OS version: build 16299.371
Browser: Microsoft edge browser
Edge version: Microsoft Edge 41.16299.371.0

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: (if applicable)

#### Describe the issue:

Keyboard focus is overlapping with content of the link. As shown attached screenshot char n is being cut.

#### Actual behavior:

Keyboard focus is overlapping with content of the link. As shown attached screenshot char n is being cut.

#### Expected behavior:

There should be some gap between the focus ring and the text.

### If applicable, please provide a codepen repro:

![image](https://user-images.githubusercontent.com/26891187/40361718-e9bdb654-5de7-11e8-8772-defb6318e008.png)

**Comment by natalieethell** [comment]
Hey @lovakumar! This issue was recently fixed in @JasonGore's PR: #5859. I'll close out this issue.
